src/pages/about.js

```diff
@@ -24,75 +29,24 @@ export default function About () {
-            </span>
-            {' '}
-            dictum  varius duis at. Hendrerit dolor magna eget est lorem ipsum dolor. Maecenas pharetra convallis posuere morbi leo urna molestie at elementum. Pharetra diam sit amet nisl. Tristique et egestas quis ipsum suspendisse ultrices gravida dictum. Non curabitur gravida arcu ac tortor dignissim. Pellentesque id nibh tortor id. Ac turpis egestas maecenas pharetra. In massa tempor nec feugiat. In eu mi bibendum neque egestas congue quisque egestas diam. Habitant morbi tristique senectus et netus et. Sit amet nulla facilisi morbi. Imperdiet dui accumsan sit amet. Commodo nulla facilisi nullam vehicula ipsum a. Pharetra et ultrices neque ornare.
-          </p>
-          <p>
-            Enim blandit volutpat maecenas volutpat blandit aliquam etiam. Suspendisse
-            {' '}
-            <span
-              onClick={togglePause}
-              onMouseEnter={() => setIsPaused(false)}
-              onMouseLeave={() => setIsPaused(true)}
-              style={{ color: 'teal', fontWeight: 'bold', textDecoration: 'underline' }}
-            >
-              interdum
-            </span>
-            {' '}
-            consectetur libero id faucibus nisl. Enim diam vulputate ut pharetra sit amet. Vitae aliquet nec ullamcorper sit amet risus. Et netus et malesuada fames ac turpis. Volutpat blandit aliquam etiam erat velit scelerisque. Condimentum mattis pellentesque id nibh tortor. Lacinia quis vel eros donec ac odio tempor orci dapibus. Rutrum tellus pellentesque eu tincidunt tortor aliquam. Imperdiet dui accumsan sit amet nulla. Duis ut diam quam nulla porttitor. Sed vulputate mi sit amet mauris commodo quis.
-          </p>
-          <p>
-            Tincidunt tortor aliquam nulla facilisi. Eu lobortis elementum nibh tellus
-            {' '}
-            <span
-              onClick={togglePause}
-              onMouseEnter={() => setIsPaused(false)}
-              onMouseLeave={() => setIsPaused(true)}
-              style={{ color: 'teal', fontWeight: 'bold', textDecoration: 'underline' }}
-            >
-              molestie
-            </span>
-            {' '}
-            nunc.  Morbi tristique senectus et netus et malesuada fames ac. Turpis massa sed elementum tempus. Non quam lacus suspendisse faucibus interdum posuere lorem ipsum dolor. Dignissim diam quis enim lobortis scelerisque fermentum. Viverra maecenas accumsan lacus vel facilisis volutpat est velit egestas. Mattis ullamcorper velit sed ullamcorper morbi tincidunt. Dapibus ultrices in iaculis nunc sed augue lacus. Sollicitudin aliquam ultrices sagittis orci a. Dui sapien eget mi proin sed libero enim. Lectus nulla at volutpat diam ut venenatis. Tellus cras adipiscing enim eu turpis egestas pretium aenean. Vitae et leo duis ut diam quam nulla porttitor massa. Tristique risus nec feugiat in. Quam elementum pulvinar etiam non. Diam maecenas sed enim ut sem. Sodales ut eu sem integer vitae justo eget magna fermentum.
```

Comment:
I deleted a bunch of this noise. All this additional content was useless and obscured the sticky footer fixes from my last PR

![image](https://github.com/peterjmartinson/huygens-site/assets/17347977/09ccc92f-c6c8-40b1-b9aa-184dc7e5727e)